### PR TITLE
Aug 2023 refresh

### PR DESCRIPTION
**Pull request by James-cloudcannon**
- [x]  Modernize the app user experience to enhance usability and user satisfaction.
- [x]  Implement a draft option for blogs to enable content creators to work on articles before publishing.
- [x]  Introduce optional dropdowns for navigation menus to improve navigation flexibility.
- [x]  Address and fix bugs reported in the current templates.
- [x]  Correct any typographical errors found in the templates.
- [x]  Fix broken scripts that are causing errors or affecting functionality.
- [x]  Increase SEO

### CODE REVIEW

File: cloudcannon.config.yaml [hunk 1]
```diff
@@ -1,11 +1,70 @@
...
+      post: 
+        path: schemas/post.md
+    add_options:
+      - name: Add Article
+        editor: content
+        schema: post
     _enabled_editors:
       - content
       - visual
+  meta_data:
```
Comment (NJKode) [inline cloudcannon.config.yaml]: Metadata is one word, so this should be `metadata` imo

Comment (NJKode) [reply cloudcannon.config.yaml]: Or just `data`

File: cloudcannon.config.yaml [hunk 2]
```diff
@@ -33,12 +92,106 @@ _inputs:
...
+    type: select
+    options:
+      values: 
+        - website
+        - article
+        - profile
+  favicon_icon:
+    type: image
+    comment: For the shortcut icon meta tag
+  faivcon_image:
```
Comment (NJKode) [inline cloudcannon.config.yaml]: typo here

File: data/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+BaseURL: "http://example.org"
```
Comment (NJKode) [inline data/meta.yaml:1]: Is this a duplication of `baseURL` in `config.toml`? Could be confusing to know which one to update

File: data/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+site_title: "Sendit"
```
Comment (NJKode) [inline data/meta.yaml:2]: Ditto with `title` in `config.toml`

File: content/blog.md
Comment (NJKode) [inline content/blog.md:1]: What's the rationale behind the name change? From my understanding, the `blog` section needs an `_index.md` file to be properly considered a section.

**Review by NJKode — CHANGES_REQUESTED** [review]
Coupla comments in here, looks pretty nice though!

Also on a general note:
* The site icon should be updated to the one that is used on all the other versions (i.e. the red one)
* The blog heading says "Blogs" instead of "Blog"

File: cloudcannon.config.yaml (hunk 1 above)
Comment (James-cloudcannon) [reply cloudcannon.config.yaml]: i'll rename this to data

**Review by NJKode — APPROVED** [review]

**Pull request merged**